src/state-indicators.service.js

```diff
@@ -24,13 +24,15 @@ class StateIndicatorsService {
     STOCK_STATUSES,
     lgasService,
     statesService,
-    thresholdsService
+    thresholdsService,
+    productListService
```

Comment:
Remember to inject it here also https://github.com/fielded/angular-nav-state-indicators/blob/master/src/state-indicators.service.js#L138